providers/fab/provider.yaml

```diff
@@ -56,6 +56,13 @@ config:
   fab:
     description: This section contains configs specific to FAB provider.
     options:
+      access_denied_message:
+        description: |
+          The message displayed when a user attempts to execute actions beyond their authorised privileges.
+        version_added: 2.0.3
```

Comment:
I assume the next fab provider version is `2.0.3` should this be manually set here?

Reply:
That's automatically done in the release process I think, at least it was part of the core release process as I remember it so I assume there's something similar for providers.